net.sourceforge.scidvspc.scidvspc.yml

```diff
@@ -7,10 +7,10 @@ rename-icon: scid #128x128
 
 finish-args:
   - --filesystem=xdg-documents
-  - --share=network     
-  - --socket=x11           
-  - --share=ipc         
-  - --device=dri         
+  - --share=network
+  - --socket=fallback-x11
```

Comment:
If I add `--socket=wayland`, the program won't start. But if I only specify `fallback-x11`, the program can start as a wayland app. I'm not sure what to do here.

Reply:
As far as I understand tk/tcl does not support Wayland.

Reply:
As far as I understand, tk/tcl doesn't support Wayland. I assume that by using "--socket=fallback-x11", you're using the X11 integration that Flatpak offers in Wayland.

The program crashes if "--socket=x11" is specified on your PC?

Reply:
I agree -- actually it turns out something's strange about my system, where I can run scid vs pc (either with or without flatpak) native on Wayland. Thanks for looking into this!

Reply:
Yes, it's magic.